### DISCUSSION TITLE
Option to load all sidebar on browser launch

### DISCUSSION THREAD

**Discussion by nexus-337** [Floorp-Projects/Floorp#892] [Feature Request]
I mainly use this browser for its  sidebar feature. but after I restart the browser, the sidebar pages don't load until I click on them. Its annoying, especially when in a hurry to get to something like WhatsApp Web . It'd be awesome if there was an option to have those sidebar pages load up as soon as I open the browser.

**Comment by Telmesomething** [comment]
You could set all the sites you want to open on startup as your home page. The principle is the same as described in this [Firefox ](https://support.mozilla.org/en-US/kb/how-to-set-the-home-page#w_set-multiple-pages-as-your-home-page)document.

**Reply by nexus-337** [reply]
but it would open the pages as tabs on startup right, i didnt want that . i wanted it to be loaded in the sidebar , not as a tab

**Reply by Telmesomething** [reply]
It doesn't appear to be possible at this moment in time, or at least I can't find anything in the [Official Documentation](https://docs.floorp.app/en/introduction/#_top). 

That said, Floorp is still in the throes of growing up so to speak and changes will no doubt appear in good time. At the moment though lack of funds seem to be the main problem and the project needs more financial support.

**Comment by Telmesomething** [comment]
There are some new additions to Floorp in the latest version [11.10.2](https://blog.ablaze.one/3992/2024-02-22/) which might be relevant to your needs.
